src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/M2FiltersArgumentCollection.java

```diff
@@ -48,7 +44,7 @@
     public int maxMedianClippingDifference = 1;
 
     @Argument(fullName = "maxMedianFragmentLengthDifference", optional = true, doc="filter variants for which alt reads' median fragment length is very different from the median for ref reads.")
```

Comment:
The old value was filtering out a lot of good variants.  I made a mistake setting it this low.  This is the value I used in *all* of our validations.